main.go

```diff
@@ -151,6 +154,19 @@ func connectAPI(addr string, token string) (api.FullNode, jsonrpc.ClientCloser,
 
 	srv.Logger.Info("Connected to Lotus version: ", version.String())
 
+	c, _ := semver.NewConstraint(">= 1.5")
```

Comment:
This issue was very old...
I think what we eventually want is to avoid incorrect matches.. 
we should later make this a bit more flexible and make the 1.5 configurable
but we can leave like this for now

Reply:
Using `NewConstraint` should let us to pass version constraints at build. For now it is hardcoded so I don't check the error but we could easily have that.